WooCommerce/Classes/ViewRelated/Products/Filters/FilterListViewController.xib

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
it's tricky because the container wouldn't know the height of the content beforehand. do you have better ideas than adding a "fake" constraint that is removed at build time (an option in the xib)?

Reply:
I added a temporary height constraint to the bottom container view that is removed at build time https://github.com/woocommerce/woocommerce-ios/pull/2204/commits/bef0794aef1dd14b71ef340ebdc8563267eaa69b, lemme know if you have other ideas!

Reply:
It's ok for me 👍 Another way can be adding a height constraint with a super low priority.